botocore/model.py

```diff
@@ -267,6 +269,18 @@ def enum(self):
         return self.metadata.get('enum', [])
 
 
+StaticContextParameter = namedtuple(
+    'StaticContextParameter', ['name', 'value']
+)
+
+
+ContextParameter = namedtuple('ContextParameter', ['name', 'member_name'])
+
+ClientContextParameter = namedtuple(
+    'ClientContextParameter', ['name', 'type', 'documentation']
+)
+
+
```

Comment:
i personally prefer the syntax of `typing.NamedTuple` and I think @nateprewitt just merged a minor PR that used it to replace a data class so maybe lets switch for consistency.